packages/react-components/react-popover/src/components/Popover/usePopover.ts

```diff
@@ -121,7 +121,10 @@ export const usePopover_unstable = (props: PopoverProps): PopoverState => {
 
   React.useEffect(() => {
     if (open && positioningRefs.contentRef.current) {
-      const firstFocusable = findFirstFocusable(positioningRefs.contentRef.current);
+      const containerTabIndex = positioningRefs.contentRef.current.getAttribute('tabIndex') ?? undefined;
```

Comment:
hah, I had the same thought process as Ling -- forgot that JS returns false for `isNaN(null)` 😄